app/frontend/src/views/Admin.vue

```diff
@@ -0,0 +1,41 @@
+<template>
+  <BaseSecure :admin="true">
```

Comment:
I'm "pretty" sure if you can just do the following and it'll still pass in admin as true (no v-bind required):
```html
<BaseSecure admin>
```